main.tf

```diff
@@ -14,8 +14,57 @@
  * limitations under the License.
  */
 
-resource "google_storage_bucket" "main" {
-  project  = var.project_id
-  name     = var.bucket_name
-  location = "US"
+# Create Tags Key
+
+resource "google_tags_tag_key" "key" {
+  parent       = var.tag_for == "organization" ? "organizations/${var.org_id}" : var.tag_for == "project" ? "projects/${var.project_number}" : null
```

Comment:
Can you make this variable a bool and make this assignment something like:

parent       = var.tag_for_org  ? "organizations/${var.org_id}" : "projects/${var.project_number}"

Reply:
Not an expert in Tags but is there a possibility that tags could have folders as parents (now or in the future)? If so keeping `tag_for` could be better and which also aligns with [org policy variables](https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/variables.tf#L17).